aquasec-server/templates/gate-deployment.yaml

```diff
@@ -0,0 +1,94 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-gate
+  labels:
+    app: {{ .Release.Name }}-gate
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  replicas: {{ .Values.gate.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}-gate
+      name: {{ .Release.Name }}-gate
+    spec:
+      serviceAccount: {{ .Release.Name }}-account
+      containers:
+      - name: gate
+        image: "{{ .Values.imageCredentials.repositoryUriPrefix }}/{{ .Values.gate.image.repository }}:{{ .Values.gate.image.tag }}"
+        imagePullPolicy: "{{ .Values.gate.image.pullPolicy }}"
+        env:
+        # TODO: What default value should we use for SCALOCK_GATEWAY_PUBLIC_IP
+        - name: SCALOCK_GATEWAY_PUBLIC_IP
```

Comment:
@oranmoshai can you please verify if we must specify this env var, and if not, what are the implications.

Reply:
Yes it is must for kubernetes you need to add the service name of the gateway deployment.